packages/calcite-components/src/components/rating/rating.scss

```diff
@@ -70,7 +77,7 @@
 
 .hovered,
```

Comment:
Is `.hovered` still applicable here? It seems like it might be overridden [above](https://github.com/Esri/calcite-design-system/pull/11469/files#diff-4ec2b699ba01f57ba793f63f7f60e950cd2a8bb0ba9c992e8cea8cb3de1bc25aR62-R64).